### PR TITLE
Adopt code to separated index for Snapshot

### PR DESCRIPTION
**Pull request by kpudlik**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ provider:
   region: us-west-1
   stage: ${self:custom.stage}
   environment:
-    ELASTIC_SEARCH: ${self:custom.env.elasticsearch}
+    OPENSEARCH_NODE: ${self:custom.env.elasticsearch}
```
Comment (marcinwadon) [inline serverless.yml:13]: env name should be `opensearch` as well 🤔

Comment (kpudlik) [reply serverless.yml:13]: fair point!

File: src/opensearch.ts [hunk 2]
```diff
@@ -1,159 +1,278 @@
...
+      'hash',
+      'height',
+      'subHeight',
+      'timestamp',
+      'blocks',
+    ];
+    const osSearch = getSnapshotQuery<OpenSearchSnapshot>(
+      predicateName,
+      predicateValue,
+      []
```
Comment (kriscieslak) [inline src/opensearch.ts:60]: Shouldn't be here `outerIncludes` not empty array?

Comment (kpudlik) [reply src/opensearch.ts:60]: you are right, I missed that.

**Review by marcinwadon — APPROVED** [review]

File: src/model.ts [hunk 3]
```diff
@@ -1,78 +1,82 @@
...
+};
 
-export type Balance = Ordinal & BalanceValue
+export type Balance = Ordinal & BalanceValue;
 
 type Height = {
-    min: number,
-    max: number
-}
+  min: number;
```
[the remark this answers is not in the document]
Comment (kpudlik) [reply src/model.ts:29]: I will clean the old code after implementing all the endpoints

**Pull request merged**